x-pack/plugins/ml/public/maps/util.ts

```diff
@@ -109,6 +109,34 @@ export function getInitialAnomaliesLayers(jobId: string) {
   return initialLayers;
 }
 
+export function getInitialSourceIndexFieldLayer(sourceIndicesWithGeoFields: {
```

Comment:
```suggestion
export function getInitialSourceIndexFieldLayers(sourceIndicesWithGeoFields: {
```

Reply:
Updated in 194d15241a2663884443d87ef1f6505302ba654d